crates/pallet-domains/src/lib.rs

```diff
@@ -2770,7 +2770,18 @@ impl<T: Config> Pallet<T> {
         false
     }
 
+    /// The external function used to access the extrinsics shuffling seed stored in
+    /// `BlockInvalidInherentExtrinsicData`.
```

Comment:
How about renaming `BlockInvalidInherentExtrinsicData` to `BlockInherentExtrinsicData` since this is not only used in FP but also in normal block production.

Reply:
Done!